backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js

```diff
@@ -0,0 +1,29 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const docClient = new dynamodb.DocumentClient();
+const { USER_TABLE } = require('../../constants');
+
+/**
+ * Creates a new user in dynamodb after they confirm their email.
+ */
+exports.cognitoPostConfirmationHandler = async (event, context, callback) => {
+    console.log('received: ', event);
+    
+    const params = {
+        Item: {
+            "pk": event.request.userAttributes.sub,
+            "name": event.request.userAttributes.name + " " + event.request.userAttributes.family_name,
+            "email": event.request.userAttributes.email,
+            "profilePicture":"",
+            "businessOwner": false,
+            "businessId": ""
+        },
+        ReturnConsumedCapacity: "TOTAL",
+        TableName: USER_TABLE
+    };
+    await docClient.put(params, function(err, data) {
+        if (err) console.log(err, err.stack);
+        else console.log(data);
```

Comment:
I don't think you need to pass a function here since you're already calling .promise()

Reply:
I'll test this in the console to make sure it works and then make the change.